bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -3488,7 +3489,9 @@ public DataSource getDataSource(long objectId) throws TskDataException, TskCoreE
 	 * @throws TskCoreException
 	 */
 	public ArrayList<BlackboardArtifact> getBlackboardArtifacts(int artifactTypeID) throws TskCoreException {
-		return getArtifactsHelper("blackboard_artifacts.artifact_type_id = " + artifactTypeID);
+		ArrayList<BlackboardArtifact> artifacts = new ArrayList<>();
```

Comment:
Can this method be deprecated with a referral to an equivalent Blackboard method?